backend/src/main/java/ai/verta/modeldb/versioning/blob/container/PathDatasetComponentContainer.java

```diff
@@ -0,0 +1,24 @@
+package ai.verta.modeldb.versioning.blob.container;
+
+import ai.verta.modeldb.ModelDBException;
+import ai.verta.modeldb.versioning.autogenerated._public.modeldb.versioning.model.AutogenGitCodeBlob;
+import ai.verta.modeldb.versioning.autogenerated._public.modeldb.versioning.model.AutogenPathDatasetComponentBlob;
+import io.grpc.Status.Code;
+
+public class PathDatasetComponentContainer implements BlobContainerBase {
```

Comment:
Why do we need a container wrapping around the autogen functionality? Seems like a good amount of boilerplate that the autogen was created to remove.

Reply:
because I thought that we need to check a and b only if it should present in diff. But now I think that it is not so important. Fixed.